### DISCUSSION TITLE
Catchment Fortuna-2.5 (CLSM-F2.5) LSM

### DISCUSSION THREAD

**Discussion by Gayathridevi123** [NASA-LIS/LISF#1009] [Q&A] [answered]
Hi David,

I would like to try with the Catchment Fortuna-2.5 (CLSM-F2.5) LSM over the following domain at 0.25 deg resolution. 

Map projection of the LIS domain:   latlon
Run domain lower left lat:            6.375
Run domain lower left lon:           66.375
Run domain upper right lat:         38.375
Run domain upper right lon:        99.875
Run domain resolution (dx):          0.25
Run domain resolution (dy):          0.25

Downloaded the CLSMF parameters from https://portal.nccs.nasa.gov/lisdata_pub/data/PARAMETERS/clsm25_parms/GLDAS_0.25-deg/. I don't know what are all these parameter files (CLSMF25 tile coordinate file, tile veg file, topo ar file, topo bf, etc). 

LDT was giving an error:-

 [INFO] Reading landmask ./input/LS_PARAMETERS/clsmf25_parms/CLSMF/clsm25_parms/GLDAS_0.25-deg/landmask_mod44w_025.1gd4r (           0 )
 [ERR] The number of *subsetted* row or column points
           EXCEEDS the total *global* row or column points
           for the input parameter file, respectively.
 
   - The subsetted, global row points ::         129 >         109
 
   Please check your specified upper-right corner longitude and
    latitude extents, especially if you have entered a gridcell
    resolution that does not divide *nicely* into a 1-deg gridcell.
   Program stopping ...
 endrun is being called

Here attaching the ldt.config and log file. 
[ldt1.txt](https://github.com/NASA-LIS/LISF/files/8212505/ldt1.txt)
[ldtlog1.txt](https://github.com/NASA-LIS/LISF/files/8212506/ldtlog1.txt)

Can you please help me with this and give me valuable suggestions? 

Thank you 
Gayathri

**Comment by dmocko** [comment, marked as answer]
Hi Gayathri,

I looked at your ldt.config file, and see the following changes that are needed to get LDT to run:

1. All of these parameter datasets are global (to 60 South).  So, despite your "Run domain" being set as it is, you need to change the lower/upper left/right lat/lon values to the same global domain for these parameters: Landmask, CLSMF25, Soils.
2. For `Landcover upper right lat:`, you have the value of 89.8755.  It should be just 89.875.
3. You need to use the U.S. spelling of "neighbor".  Change from "neighbour".

The corrected lines in your ldt.config file should be:

```
Landcover fill option:           neighbor
Landcover upper right lat:       89.875

Landmask spatial transform:     neighbor
Landmask lower left lat:      -59.875
Landmask lower left lon:     -179.875
Landmask upper right lat:      89.875
Landmask upper right lon:     179.875

CLSMF25 spatial transform:          neighbor
CLSMF25 lower left lat:            -59.875
CLSMF25 lower left lon:           -179.875
CLSMF25 upper right lat:            89.875
CLSMF25 upper right lon:           179.875

Soils spatial transform:      neighbor
Soils lower left lat:        -59.875 
Soils lower left lon:       -179.875
Soils upper right lat:        89.875
Soils upper right lon:       179.875

Topography spatial transform:   neighbor

Albedo spatial transform:       neighbor

Greenness spatial transform:     neighbor

LAI/SAI spatial transform:       neighbor

```

**Reply by Gayathridevi123** [reply]
Hi David,

Thank you so much for your help. After the changes were made, LDT and LIS ran successfully. The parameter dataset that I'm using is global and has 0.25 deg resolution.  

**Have these parameter datasets are available at 1KM resolution over the global domain?** Looking forward to hearing from you.

Kind regards,
Gayathri

**Reply by dmocko** [reply]
No, these parameter datasets are not available globally at 1km.

David

**Reply by Gayathridevi123** [reply]
Thank you David for your reply.
